modules/developers-guide/pages/work-with-languages.adoc

```diff
@@ -161,29 +161,29 @@ service : {
 ----
 . Save your changes and close the file to continue.
 
-=== Modify the default program
+=== Modify the default dapp
 
-When you create a new project, your project `+src+` directory includes a template `+main.rs+` file with the "Hello, World!" program.
+When you create a new project, your project `+src+` directory includes a template `+main.rs+` file with the "Hello, World!" dapp.
 
 To modify the template source code:
 
 . Open the template `+src/main.rs+` file in a text editor and delete the existing content.
-. Write the program you want to deploy on the {IC}.
+. Write the dapp you want to deploy on the {IC}.
 +
-As you write your program, keep in mind that there are two types of calls—update calls and query calls—and that update functions use asynchronous messaging.
+As you write your dapp, keep in mind that there are two types of calls—update calls and query calls—and that update functions use asynchronous messaging.
 . Save your changes and close the `+main.rs+` file.
 
 === Connect to a network and deploy
 
-Before you can deploy and test your program, you need to do the following:
+Before you can deploy and test your dapp, you need to do the following:
 
-* Connect to the {IC} network either running locally in your development environment or running remotely on a subnet that you can access.
+* Connect to the {IC} network either running locally in your development environment or running remotely (on-chain) on a subnet that you can access.
```

Comment:
"local canister execution environment" vs "IC blockchain mainnet" or any accepted abbreviation for the second :)